### ISSUE TITLE
Implement and ingest BOX logs via a custom ingester

### ISSUE THREAD

**Issue by lota-1234** [pipelinelabo/DatalaiQ#5]

**Comment by lota-1234** [comment]
@ppln-allanw  san, discussed with Anis Bhaia, he told, he has made some progress. but we need the document from him for where we can engage Mushfiq for better potential

**Comment by ppln-allanw** [comment]
Are you asking me to build a document?

**Comment by lota-1234** [comment]
No, I am informing you that we need a document from Anis Bhaia.
